File: Obj-C & Swift 3 - Getting Started/Loops/Loops/ViewController.m
//
//  ViewController.m
//  Loops
//
//  Created by Jeffrey Eng on 4/20/17.
//  Copyright © 2017 Jeffrey Eng. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    NSArray *cars = @[@"Bimmer", @"Maserati", @"Bronco II"];
    
    // Standard for-loop
    for (int x = 0; x < cars.count; x++) {
        NSString *car = [cars objectAtIndex:x];
        NSLog(@"Car: %@", car);
    }
    
    // Fast enumeration / for each loop
    for (NSString *car in cars) {
        NSLog(@"Car: %@", car);
    }
    
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: Building Full Stack Apps with Firebase/pushMyNotifs/pushMyNotifs/ViewController.swift
//
//  ViewController.swift
//  pushMyNotifs
//
//  Created by Jeffrey Eng on 2/20/17.
//  Copyright © 2017 Jeffrey Eng. All rights reserved.
//

import UIKit
import Firebase
import FirebaseInstanceID
import FirebaseMessaging

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
       
        FIRMessaging.messaging().subscribe(toTopic: "/topics/news")
        
    }

    
    
}

